test/integration/targets/cloudformation_exports/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: set connection information for aws modules and run tasks
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+
+  block:
+    - name: Create a minimal stack with an export set by parameter
+      cloudformation:
+        stack_name: "{{ stack_name }}"
+        template_body: "{{ lookup('file','test_stack.yml') }}"
+        template_parameters:
+          TestParamName: "cf-exports-param"
+          TestParamValue: "Set By CF Exports"
+          BucketSuffix: "{{ resource_prefix }}"
+      register: cf_stack
+    - name: Read from Exports
+      cloudformation_exports:
+        region: "{{ aws_region }}"
+      register: exports_result
+    - set_fact:
+        export_value: "{{ exports_result['export_items']['cf-exports-param'] }}"
+    - debug:
+        var: export_value
+    - assert:
+        that:
+          - export_value is defined
```

Comment:
We should try to assert some additional things about the export_value, like `export_value['Name'] is "Set By CF Exports"` and the length/number of exports.